Source/WebCore/dom/SlotAssignment.cpp

```diff
@@ -530,9 +553,13 @@ void ManualSlotAssignment::hostChildElementDidChangeSlotAttribute(Element&, cons
 {
 }
 
-void ManualSlotAssignment::willRemoveAssignedNode(const Node&)
+void ManualSlotAssignment::willRemoveAssignedNode(const Node& node, ShadowRoot& shadowRoot)
 {
     ++m_slottableVersion;
+    if (RefPtr slot = node.assignedSlot()) {
+        if (slot->containingShadowRoot() == &shadowRoot)
```

Comment:
nit: Could be on one line:
`if (RefPtr slot = node.assignedSlot(); slot->containingShadowRoot() == &shadowRoot) {`

Reply:
Nice. Done.

Reply:
Actually, I need to check the nullity of `slot` but that works.

Reply:
Oh totally, my bad. I sometimes make this mistake :|